pulp_smash/constants.py

```diff
@@ -202,6 +202,11 @@
     http://docs.pulpproject.org/plugins/pulp_ostree/tech-reference/importer.html
 """
 
+OSTREE_FEED_SMALL = (
+    'https://repos.fedorapeople.org/pulp/pulp/fixtures/ostree/small/'
```

Comment:
Is this going to be changed? I think so, but it may be useful to comment here so others know as well.